frontend/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,11 @@
+import { AiOutlineSend } from 'react-icons/ai';
+import { StyledButton } from 'components/Button/Button.styled';
+
+export const Button = ({ callback, buttonName }) => {
+  return (
+    <StyledButton onClick={callback}>
+      {buttonName}
+      <AiOutlineSend />
+    </StyledButton>
+  );
+};
```

Comment:
Nie zawsze button będzie miał ikonkę, i nie zawsze tą samą. Możliwe, że będziemy chcieli dodać własną albo nie pokazywać jej. Dasz radę to zaimplementować? Wykorzystaj  "children" aby przekazać ikonę z zewnątrz do komponentu

Reply:
Oki

Reply:
![image](https://user-images.githubusercontent.com/22809303/151062552-6573d6eb-20f7-45fc-92f4-6da48f8cece9.png)
@PiotrRynio  to masz na myśli?

Reply:
TAK!!!
Myślałem o czymś innym, ale to Twoje to o wiele lepsze rozwiązanie :)